src/main/kotlin/com/team4099/robot2022/commands/intake/IntakeCommand.kt

```diff
@@ -0,0 +1,22 @@
+package com.team4099.robot2022.commands.intake
+
+import com.team4099.lib.logging.Logger
+import com.team4099.robot2021.subsystems.com.team4099.robot2022.subsystems.Intake
+import com.team4099.robot2022.config.Constants
+import edu.wpi.first.wpilibj2.command.CommandBase
+
+class IntakeCommand(
```

Comment:
does this need to exist if we have commands for every combination we actually expect to use?